src/styles/base/_focus.scss

```diff
@@ -1,40 +1,40 @@
-@mixin focus-default
-{
+@mixin focus-default {
+  box-shadow: none;
+
   // Use for most interactive elements on a light background
   outline: space(xs) solid $action-primary-40;
   outline-offset: space(xs); // NOTE: this is not supported in IE, but it will degrade gracefully
-  box-shadow: none;
 }
 
-@mixin focus-default-inset
-{
+@mixin focus-default-inset {
+  box-shadow: inset 0 0 0 space(xs) $action-primary-40;
+
   // Use for interactive elements on a light background that are contained in wrappers that would conceal the default focus indicator
   outline: 0;
```

Comment:
these mixins got weird. can we adjust whatever rule/s is/are moving rules above the descriptive comment and adding weird space in between?